src/Evolution/Executables/GrMhd/ValenciaDivClean/CMakeLists.txt

```diff
@@ -20,34 +21,97 @@ set(LIBS_TO_LINK
   )
 
 add_spectre_parallel_executable(
```

Comment:
would it not be possible to have a CMake list of all the ones to generate and then do a for loop over the list? That seems more maintainable than the current thing done here.

Reply:
I can make a function that takes two arguments, but the arguments (the post-Evolve part of the name of the executable, and the template parameter for `EvolutionMetavars`) cannot be constructed from one list...

Reply:
Hmm, so we could have two lists, or a list with "pairs", e.g. `"namespace::A,EvolveA;namespace::B,EvolveB;namespace::C,EvolveLarry"`. Thoughts?